SCClassLibrary/Common/Collections/Scale.sc

```diff
@@ -19,8 +19,13 @@ Scale {
 	}
 
 	*doesNotUnderstand { |selector, args|
-		var scale = this.newFromKey(selector, args).deepCopy;
-		^scale ?? { super.doesNotUnderstand(selector, args) };
+	    var scale = this.newFromKey(selector, args);
+        if(scale.notNil) { ^scale };
```

Comment:
just indentation

Reply:
Seems I've used spaces instead of tabs somehow... I'll fix shortly.